www/src/components/clusters/Clusters.js

```diff
@@ -49,6 +49,21 @@ function DataEntry({ name, value, children }) {
 }
 
 function ClusterDescription({ q }) {
+  function DomainLink() {
+    if (q.domain == null || q.domain === '') return (<i>Empty</i>)
```

Comment:
this clause will technically never hit because of api validation but it doesn't hurt to be safe